docs/docs/the-actions.md

```diff
@@ -276,13 +276,30 @@ const getNew = () => {
 
 # Clearing state
 
+## Reset State
+
+**Format:** `([object]) => void`
+
+In order to reset a single `state.list` to its initial state, you can pass it's name as `options` to `resetState`.
+
+```
+const [, { resetState }] = useCroods({ name: 'todos' })
+const clearAll = () => {
+  myApi.removeUser().then(() => resetState({ name: 'todos' }))
+}
+```
+
+This will clear the state back to its initial values, with every Croods information inside it. The initial state has empty `info` and `list`.
```

Comment:
I removed the "empty" bit because it was wrong, the info is `null` but the link I added clarifies it and we avoid having to maintain this bit of docs if we ever change it
```suggestion
This will clear the state back to its [initial values](/docs/main-concepts#state), with every Croods information inside it.
```